lib/scholar/integrate/simpson.ex

```diff
@@ -0,0 +1,446 @@
+defmodule Scholar.Integrate.Simpson do
```

Comment:
LGTM! We can keep the `Scholar.Integrate.Simpson` private (with `@moduledoc false`) and then we move the `deftransform simplson` to `Scholar.Integrate` and have it call `Scholar.Integrate.Simpson.simpson_n`.

Reply:
Yeah, that's really good idea to me

Reply:
I think that now, we can move both Simpson and trapezoidal to `integrate.ex` and have general interface `Scholar.Integrate.integrate` and more specific calls like `Scholar.Integrate.simpson`. This will save all the documentation.